client/src/Features/AssignmentLinks/AssignmentLinkEditor.tsx

```diff
@@ -34,12 +50,15 @@ const AssignmentLinkEditorInner = ({
         value={link?.displayText || ''}
         styles={themedClassNames(textFieldBaseStyle)}
         onChange={(_e, newValue) => onChangeTextField('displayText', newValue || '')}
+        onGetErrorMessage={ValidateTitleLength}
       />
       <TextField
         value={link?.url || ''}
         placeholder="URL"
         styles={themedClassNames(urlInputStyle)}
         onChange={(_e, newValue) => onChangeTextField('url', newValue || '')}
+        onGetErrorMessage={ValidateURL}

```

Comment:
Just wondering if there's a way to simulate the following inside `<TextField {...props} />`

> **Reference**: https://www.w3schools.com/tags/att_input_type_url.asp

Reply:
I began exploring this itself before beginning the task if we could find some way to avoid writing the validation checks by simply adding a type field as `url`. Unfortunately, couldn't find a parallel property that could serve the purpose for the `TextField`.